application/packages/python/ftrs_data_layer/models.py

```diff
@@ -170,6 +171,71 @@ class Telecom(BaseModel):
     web: str | None
 
 
+class OpeningTimeCategory(str, Enum):
+    availableTime = "availableTime"
+    availableTimeVariations = "availableTimeVariations"
+    availableTimePublicHolidays = "availableTimePublicHolidays"
+    notAvailable = "notAvailable"
+
+
+class DayOfWeek(str, Enum):
```

Comment:
```suggestion
class DayOfWeek(str, Enum):
    MONDAY = "mon"
    TUESDAY = "tue"
    WEDNESDAY = "wed"
    THURSDAY = "thu"
    FRIDAY = "fri"
    SATURDAY = "sat"
    SUNDAY = "sun"
```

to follow the Python's convention for constants and enum members to be UPPERCASE

Reply:
done